dev-docs/source/GettingStartedWithPyCharm.rst

```diff
@@ -10,11 +10,11 @@ This tutorial assumes you are familiar with the process of building Mantid (with
 .. contents::
   :local:
 
+.. _setting-up-pycharm-on-windows:
+
 Setting up PyCharm on Windows
 #############################
 
-PyCharm should be opened using ``pycharm.bat`` which can be found in the build directory (this sets some additional environment variables compared with simply opening PyCharm directly).
```

Comment:
Can we remove ` mantid/buildconfig/windows/pycharm.bat.in` (plus the CMake generation code) now that we don't need it anymore?

Reply:
Okay. If the setting of paths manually does prove okay in the long run then I think it would be good to just reduce the steps here and not give 2 options when it's not clear why one would pick one over the other.